frontend/package.json

```diff
@@ -270,13 +285,15 @@
     "husky": "^8.0.3",
     "i18next-parser": "^9.3.0",
     "i18next-pseudo": "^2.2.0",
-    "jasmine-core": "2.x",
-    "jest": "22.x",
-    "jest-cli": "22.x",
-    "jest-junit": "^11.1.0",
-    "jest-resolve": "^26.4.0",
+    "jest": "^30.2.0",
+    "jest-cli": "^30.2.0",
+    "jest-environment-jsdom": "^30.2.0",
+    "jest-jasmine2": "^30.2.0",
```

Comment:
Why not `jest-circus`, which appears to be the default for Jest?

Reply:
Good point. Will switch to jest-circus